src/main/java/org/broadinstitute/consent/http/models/DataSet.java

```diff
@@ -168,4 +174,10 @@ public Integer getAlias() {
     public void setAlias(Integer alias) {
         this.alias = alias;
     }
+
+    public String getAliasParsed() {return aliasParsed; }
```

Comment:
You could simplify the class a little by doing the parsing here in the getter and avoid the setter altogether:
```suggestion
    public String getAliasParsed() {
        return DatasetUtil.parseAlias(alias); 
    }
```

Reply:
Another issue we're working around is the fact that the UI refers to this value as "Dataset ID" instead of alias. Since we already have a `datasetId`, can we call this instead a `datasetIdentifier` (I'm open to other suggestions as well, i.e. `uniqueDatasetIdentifier` or anything similar that conveys the UI presentation)? I'd like to get to a place where the naming is more intuitive between this class and what we use in the display.